layouts/partials/episode-module.html

```diff
@@ -93,7 +98,7 @@ <h4>Subscribe for free!</h4>
                 'replace' below is to ensure correct local link across local, dev, and prod URLs.
                 Front matter does not allow variables, so we need to do string substitution.
             */}}
-            {{ (replace .Params.showNotes "!!baseurl!!/" .Site.BaseURL) | markdownify }}
+            {{ (replace .Params.showNotes "!!baseurl!!" .Site.BaseURL) | markdownify }}
```

Comment:
I had to remove the `/` from the string to be replaced. It was correct for local dev, but once on stage, it caused a bug. (Local dev still works like this, there's just an extra slash in the URL.)